packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2799,3 +2733,104 @@ int _toLocationId(_Location location) {
   _locationToId[location] = id;
   return id;
 }
+
+class _SerializationConfig extends DiagnosticsSerializationDelegate {
```

Comment:
nit: maybe name this _SerializationDelegate or rename the base class to
DiagnosticsSerializationConfig?

Reply:
Done